Adding Entra ID SCIM (#7413)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

- Given name

The following IdPs are supported in the <Constant name="cloud" /> UI:
- [Okta](#scim-configuration-for-okta)
- [Entra ID](#scim-configuration-for-entra-id)

> If your IdP isn’t on the list, it can be supported using <Constant name="cloud" /> APIs (_docs coming soon_).

Reviewer: API docs are up so I think we can update this note and link here: https://docs.getdbt.com/dbt-cloud/api-v3#/operations/Retrieve%20SCIM%20configuration

> If your IdP isn’t on the list, it can be supported using <Constant name="cloud" /> APIs (_docs coming soon_).

Author: ```suggestion
If your IdP isn’t on the list, it can be supported using <Constant name="cloud" /> [APIs](https://docs.getdbt.com/dbt-cloud/api-v3#/operations/Retrieve%20SCIM%20configuration).
```

2. From under **Manage**, again click **Provisioning**.
3. Expand the **Mappings** section and click **Provision Microsoft Entra ID users**.
     <Lightbox src="/img/docs/dbt-cloud/access-control/provision-entra-users.png" width="60%" title="Provision the Entra ID users." />
4. Make sure the following mappings are in place and delete any others:
    - **UserName:** `userPrincipalName`
    - **active:** `Switch([IsSoftDeleted], , "False", "True", "True", "False")`
    - **emails[type eq "work"].value:** `userPrincipalName`
    - **name.givenName:** `givenName`
    - **name.familyName:** `surname`
    - **externalid:** `mailNickname`
    - **emails[type eq "work"].primary** (see next step)
     <Lightbox src="/img/docs/dbt-cloud/access-control/attribute-list.png" width="60%" title="Edit the attributes so they match the list as shown." />
5. Click **Add new mapping** and fill out the fields with the following:
    - **Mapping type:** `none`
    - **Default value if null (optional):** `True`
    - **Target attribute:** `emails[type eq "work"].primary`
    - **Match objects using this attribute:** `No`
    - **Matching precedence:** *Leave blank*
    - **Apply this mapping:** `Always`

>     Click **Ok**

Reviewer 2: This is rolled up with the previous bullet
```suggestion
  6. Click **Ok**
```

>   6. Click **Ok**

Reviewer 2: ```suggestion
6. Click **Ok**
```

    <Lightbox src="/img/docs/dbt-cloud/access-control/edit-attribute.png" width="60%" title="Edit the attribute as shown." />

> 6. Back on the **Attribute mapping** screen, click the checkbox to **Show advanced options**

Reviewer 2: ```suggestion
7. Back on the **Attribute mapping** screen, click the checkbox to **Show advanced options**
```

    <Lightbox src="/img/docs/dbt-cloud/access-control/show-advanced-options.png" width="60%" title="Edit the advanced options." />

> 7. Mark all of the above fields as `Required` and click **Save**.

Reviewer 2: ```suggestion
8. Mark all of the fields from Step 5 as `Required` and click **Save**.
```
